shared/annotation.py

```diff
@@ -371,7 +371,7 @@ def instance_template_main(self):
 
     def __check_all_instances_available_in_new_instance_list(self):
         if not self.do_init_existing_instances:
-            return
+            return True
```

Comment:
Good catch :)

Reply:
Yes Thanks for noticing that one Anthony :)